profiles/urls.py

```diff
@@ -1,6 +1,11 @@
 from django.contrib import admin
 from django.urls import path
 
-urlpatterns = [
+from profiles.views import StartUpProfilesView, StartUpProfileCreate, StartUpProfileUpdate, StartupProfileViewById
 
+urlpatterns = [
+    path('startups/', StartUpProfilesView.as_view(), name='startup-list'),
+    path('create', StartUpProfileCreate.as_view(), name='startup-create'),
+    path('update/<int:pk>/', StartUpProfileUpdate.as_view(), name='star-update'),
```

Comment:
When defining RESTful URLs, it’s better to avoid verbs like `create` and `update` in the URL paths because HTTP methods already imply the action. Instead of `/startups/create/` and `/startups/update/<int:pk>/`, you could opt for the following:

- `POST` request to `/startups/` to create a new startup.
- `PUT` or `PATCH` request to `/startups/<int:pk>/` to update an existing startup.